provision/vvv-init.sh

```diff
@@ -1,31 +1,75 @@
 #!/usr/bin/env bash
 
-# Make a database, if we don't already have one
-echo -e "\nCreating database 'yoast_acf_analysis_development' (if it's not already there)"
-mysql -u root --password=root -e "CREATE DATABASE IF NOT EXISTS yoast_acf_analysis_development"
-mysql -u root --password=root -e "GRANT ALL PRIVILEGES ON yoast_acf_analysis_development.* TO wp@localhost IDENTIFIED BY 'wp';"
+# Variables to use for proper provisioning.
+DOMAIN=`get_primary_host "${VVV_SITE_NAME}".test`
+DOMAINS=`get_hosts "${DOMAIN}"`
+SITE_TITLE=`get_config_value 'site_title' "${DOMAIN}"`
+WP_VERSION=`get_config_value 'wp_version' 'latest'`
+WP_TYPE=`get_config_value 'wp_type' "single"`
+DB_NAME=`get_config_value 'db_name' "${VVV_SITE_NAME}"`
+DB_NAME=${DB_NAME//[\\\/\.\<\>\:\"\'\|\?\!\*-]/}
+
+# Make a database, if we don't already have one.
+echo -e "\nCreating database '${DB_NAME}' (if it's not already there)"
+mysql -u root --password=root -e "CREATE DATABASE IF NOT EXISTS ${DB_NAME}"
+mysql -u root --password=root -e "GRANT ALL PRIVILEGES ON ${DB_NAME}.* TO wp@localhost IDENTIFIED BY 'wp';"
 echo -e "\n DB operations done.\n\n"
 
 # Nginx Logs
 mkdir -p ${VVV_PATH_TO_SITE}/log
 touch ${VVV_PATH_TO_SITE}/log/error.log
 touch ${VVV_PATH_TO_SITE}/log/access.log
 
-cd ${VVV_PATH_TO_SITE}
-
-noroot composer install
-
-cd ${VVV_PATH_TO_SITE}/wp-content/plugins/yoast-acf-analysis
-noroot composer install
-noroot npm install
+# Install and configure the latest stable version of WordPress
+if [[ ! -f "${VVV_PATH_TO_SITE}/public_html/wp-load.php" ]]; then
+    echo "Downloading WordPress..."
+	noroot wp core download --version="${WP_VERSION}"
+    echo "Downloading WordPress complete"
+fi
 
-if [[ ! -d "${VVV_PATH_TO_SITE}/wp" ]]; then
-  cp ${VVV_PATH_TO_SITE}/provision/.env.js ${VVV_PATH_TO_SITE}/wp-content/plugins/yoast-acf-analysis/tests/js/system/.env.js
+if [[ ! -f "${VVV_PATH_TO_SITE}/public_html/wp-config.php" ]]; then
+  echo "Configuring WordPress Stable..."
+  noroot wp core config --dbname="${DB_NAME}" --dbuser=wp --dbpass=wp --quiet --extra-php <<PHP
+define( 'WP_DEBUG', true );
```

Comment:
Lacks

    define( 'AC_YOAST_ACF_ANALYSIS_ENVIRONMENT', 'development' );

without which the testing data won't get loaded.